server/middleware/errorHandler.js

```diff
@@ -0,0 +1,12 @@
+// SequelizeDatabaseError
+// SequelizeValidationError === undefined
+
+module.exports = (err, req, res, next) => {
+  if (err.name === 'SequelizeValidationError') {
+    res.status(400).json({ message: err.errors[0].message })
```

Comment:
hmm ini gak bijak untuk langsung messagenya err.errors[0].message, karena kalo errornya lebih dari satu kan bisa aja.
terserah mau bkin message nya jadi array of string atau string yang di join dari array message nya